draft-demarco-oauth-status-assertions.md

```diff
@@ -511,9 +512,9 @@ table below:
 | **error** | REQUIRED. The value SHOULD be assigned with one of the error types defined in {{RFC6749}}[Section 5.2](https://tools.ietf.org/html/rfc6749#section-5.2) or defined in the Section [Status Assertion Error Values](status-assertion-error-values). | {{RFC7519}} Section 4.1.7 |
 | **error_description** | OPTIONAL. Text that clarifies the nature of the error in relation to the `error` value.  | {{RFC7519}} Section 4.1.7 |
 
-## Rationale About The Unsigned Status Assertion Errors
+## Rationale about the Unsigned Status Assertion Errors
```

Comment:
not sure about this, headings should have the first letter of each word capitalized

@OR13 wdyt? ^